appveyor.yml

```diff
@@ -1,19 +1,21 @@
 # Specify version format
 version: "{build}"
 
-# Operating system (build VM template)
-os: Visual Studio 2015
-
-# build platform, i.e. Win32 (instead of x86), x64, Any CPU. This setting is optional.
-platform: 
-  - Win32
-  #- x64  Disable build for x64 machine until x64 version of dart-prerequisites is ready
+image: Visual Studio 2017
 
 # specify custom environment variables
+#
+# We generate project files for Visual Studio 14 because the boost binaries installed on the test server are for Visual Studio 14.
```

Comment:
I'm a bit confused. This line says we generate project files for Visual Studio 14, but then above and below it looks like everything is targeting Visual Studio 15. Am I misunderstanding what it's saying about generating project files?

Reply:
This is an outdated description. Let me update this accordingly. Probably we don't need this note because boost will be installed anyway as [`fcl` is dependent on it](https://github.com/Microsoft/vcpkg/blob/master/ports/fcl/CONTROL#L4).